libcodechecker/server/client_db_access_handler.py

```diff
@@ -272,8 +272,12 @@ def getRunData(self):
                 .group_by(Report.run_id) \
                 .subquery()
 
-            q = session.query(Run, stmt.c.report_count) \
-                .outerjoin(stmt, Run.id == stmt.c.run_id) \
+            q = session.query(Run, stmt.c.report_count)
+
+            if run_name_filter is not None:
+                q = q.filter(Run.name.like(run_name_filter + '%'))
```

Comment:
Maybe we shan't. :slightly_frowning_face: Infix searching could be more appropriate, especially for the naming conventions used internally. :/